checks.d/mongo.py

```diff
@@ -730,3 +733,41 @@ def check(self, instance):
                         submit_method(self, metric_name_alias, value, tags=ns_tags)
             except Exception, e:
                 self.log.warning('Failed to record `top` metrics %s' % str(e))
+
+        # Fetch information analogous to Mongo's db.getReplicationInfo()
+        localdb = cli['local']
+
+        oplog_data = {}
+
+        for ol_collection_name in ("oplog.rs", "oplog.$main", None):
+            ol_metadata = localdb.system.namespaces.find_one({"name": "local.%s" % ol_collection_name})
+            if ol_metadata is not None:
+                break
+
+        if ol_collection_name is not None:
+            try:
+                oplog_data['totalSizeMB'] = round(ol_metadata['options']['size'] / 2.0 ** 20, 2)
+
+                oplog = localdb[ol_collection_name]
+
+                oplog_data['usedSizeMB'] = round(localdb.command("collstats", ol_collection_name)['size'] / 2.0 ** 20, 2)
+
+                op_asc_cursor = oplog.find().sort("_id", pymongo.ASCENDING).limit(1)
+                op_dsc_cursor = oplog.find().sort("_id", pymongo.DESCENDING).limit(1)
+
+                try:
+                    first_timestamp = op_asc_cursor[0]['ts'].as_datetime()
+                    last_timestamp = op_dsc_cursor[0]['ts'].as_datetime()
+                    oplog_data['timeDiffSeconds'] = (last_timestamp - first_timestamp).total_seconds()
```

Comment:
`total_seconds` only exists since python 2.7, and we want to support 2.6 too (see the Travis failure on 2.6 on the `mongodb` test).

Could you handle this case with something similar to what we do [here](https://github.com/DataDog/dd-agent/blob/9f022645960f48c6dfdd3f1430974a2cc3d5ee8d/checks.d/mongo.py#L610-L616)? (and handle it in a function if it makes sense)